quasar-utils/src/main/groovy/net/corda/plugins/quasar/QuasarPlugin.groovy

```diff
@@ -42,24 +47,20 @@ class QuasarPlugin implements Plugin<Project> {
             throw new GradleException("The Quasar-Utils plugin requires Gradle $MINIMUM_GRADLE_VERSION or newer.")
         }
 
-        // Apply the Java plugin on the assumption that we're building a JAR.
-        // This will also create the "compile", "compileOnly" and "runtime" configurations.
-        project.pluginManager.apply(JavaPlugin)
-
         def rootProject = project.rootProject
-        def quasarGroup = rootProject.hasProperty('quasar_group') ? rootProject.property('quasar_group') : defaultGroup
-        def quasarVersion = rootProject.hasProperty('quasar_version') ? rootProject.property('quasar_version') : defaultVersion
-        def quasarClassifier = rootProject.hasProperty('quasar_classifier') ? rootProject.property('quasar_classifier') : defaultClassifier
-        def quasarPackageExclusions = rootProject.hasProperty("quasar_exclusions") ? rootProject.property('quasar_exclusions') : Collections.emptyList()
+        def quasarGroup = rootProject.findProperty('quasar_group')?.toString() ?: defaultGroup
+        def quasarVersion = rootProject.findProperty('quasar_version')?.toString() ?: defaultVersion
+        def quasarSuspendable = rootProject.findProperty('quasar_suspendable_annotation')?.toString()?.trim()
```

Comment:
shouldn't we have a default value for this?

Reply:
No, because the default value is Quasar's own `co.paralleluniverse.fibers.Suspendable` annotation, which Quasar uses anyway.

Reply:
👍🏿